src/el_cl_launcher.star

```diff
@@ -163,6 +170,37 @@ def launch(
             sequencer_enabled,
             sequencer_context,
         )
+        all_el_contexts.append(el_context)
+
+        new_el_context = None
+        if "builder-op" in cl_service_name:
```

Comment:
I really want to add a new field on cl / el side to indicate that it's builder / sequencer / verifier type instead of relying on just name for it

Reply:
Let's do it later when we formally submit PRs for this

Reply:
Yeah agree, right now it's just pretty hacky haha we should polish the changes when pushing upstream